mne/epochs.py

```diff
@@ -1491,6 +1493,85 @@ def get_data(self, picks=None, item=None):
         """
         return self._get_data(picks=picks, item=item)
 
+    @verbose
+    def apply_function(self, fun, picks=None, dtype=None, n_jobs=1,
+                       channel_wise=True, verbose=None, *args, **kwargs):
+        """Apply a function to a subset of channels.
+
+        The function "fun" is applied to the channels defined in "picks". The
+        data of the Epochs object is modified inplace. If the function returns
+        a different data type (e.g. numpy.complex) it must be specified using
+        the dtype parameter, which causes the data type used for representing
+        the raw data to change.
+        The Epochs object has to have the data loaded e.g. with
+        ``preload=True`` or ``self.load_data()``.
+
+        .. note:: If n_jobs > 1, more memory is required as
+                  ``len(picks) * n_times`` additional time points need to
+                  be temporaily stored in memory.
+        .. note:: If the data type changes (dtype != None), more memory is
+                  required since the original and the converted data needs
+                  to be stored in memory.
+
+        Parameters
+        ----------
+        fun : callable
+            A function to be applied to the channels. The first argument of
+            fun has to be a timeseries (numpy.ndarray). The function must
+            operate on an array of shape ``(n_times,)`` if
+            ``channel_wise=True`` and ``(len(picks), n_times)`` otherwise.
+            The function must return an ndarray shaped like its input.
+        %(picks_all_data_noref)s
+        dtype : numpy.dtype (default: None)
```

Comment:
```suggestion
        dtype : numpy.dtype
```

We don't document defaults like this. Instead, we add it to the description of the parameter (see below).